.test.sh

```diff
@@ -21,7 +21,9 @@ find src/ -name '*.pyc' -delete
 rm -f build/junit.xml
 
 # testing management commands that require a queue shared between processes
-LAX_MULTIPROCESSING=1 coverage run --source='src/' --omit='*/tests/*,*/migrations/*' src/manage.py test "$module" --no-input
+export DJANGO_SETTINGS_MODULE=core.settings
```

Comment:
require by pytest-django, I can also push this into a pytest.ini type file, but it's fine for now

Reply:
Would try to keep a manual `pytest ...` command easy to write, but your call. Could also be in `conftest.py` if there is one.

Reply:
eh, I'll leave it as-is.

welcome back! I bet you have a zillion emails